manifests/valve.pp

```diff
@@ -189,11 +191,26 @@
 
   # only run if ensure is not absent
   if ! ($ensure == absent) {
-    # run augeas with our changes
-    augeas {
-      'repose_sysconfig':
-        context => '/files/etc/sysconfig/repose',
-        changes => [ $repose_sysconfig, $saxon_sysconfig ]
+    if $::systemd == 'true' {
+      file { "/etc/systemd/system/repose-valve.service.d":
+        ensure => 'directory',
+        owner  => 'root',
+        group  => 'root',
+        mode   => '0755',
+      }
+        
+      file {"/etc/systemd/system/repose-valve.service.d/local.conf":
```

Comment:
Also, to @mwhahaha 's point the systemd::service_limits would work if you pass your contents via `$contents`.  I'm not excited about that interface because it feels kinda like a misuse or poorly implemented pattern. This should cover this file and the daemon reload for you.